clients/java/client/pom.xml

```diff
@@ -143,13 +187,132 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
-        <version>${version.plugin.surefire}</version>
         <configuration>
-          <skipTests>${skip.unit.tests}</skipTests>
           <trimStackTrace>false</trimStackTrace>
         </configuration>
       </plugin>
 
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <phase>generate-resources</phase>
+            <goals>
+              <goal>unpack</goal>
+            </goals>
```

Comment:
I am not at all sure about this, but something we should ensure here:
This plugin should really only unpack this in the test execution and not when tests are skipped, right?

Reply:
I see your point. What do you think about skipping the plugin when `skipTests` is `true`?

Reply:
That sounds like a good direction to take to me 👍